presto/eval.py

```diff
@@ -124,10 +128,8 @@ def finetune_sklearn_model(
     def _inference_for_dl(
         dl,
         finetuned_model: Union[PrestoFineTuningModel, BaseEstimator],
```

Comment:
shouldnt `CatBoostClassifier` type be part of this `Union`?

Reply:
Addressed with the `SklearnStyleClassifier`